apps/site/lib/site_web/views/schedule_view.ex

```diff
@@ -465,34 +465,12 @@ defmodule SiteWeb.ScheduleView do
   end
 
   @spec timetable_note(Conn.t() | map) :: Safe.t() | nil
-  def timetable_note(%{route: %Route{id: "CR-Fairmount"}, direction_id: 1}) do
-    content_tag :span, class: "m-timetable__note" do
-      [
-        content_tag(:span, "FRANK: ", class: "m-timetable__cell--via"),
-        content_tag(
-          :span,
-          "Operates from Forge Park. Does not serve Foxboro. "
-        ),
-        link(to: timetable_path(SiteWeb.Endpoint, :show, "CR-Franklin")) do
-          "See the Franklin Line schedule for all stops."
-        end
-      ]
-    end
-  end
-
   def timetable_note(%{route: %Route{id: "CR-Foxboro"}, direction_id: _}) do
     content_tag :div, class: "m-timetable__note" do
       [
         content_tag(:p, [
           content_tag(:strong, "Note:"),
           " Trains depart from Foxboro 30 minutes after conclusion of events"
-        ]),
-        content_tag(:p, [
-          content_tag(:strong, "Note:"),
-          " For regular weekday service to and from Foxboro please visit ",
-          link(to: timetable_path(SiteWeb.Endpoint, :show, "CR-Franklin")) do
-            "Franklin Line"
-          end
```

Comment:
Are we removing this piece because the Foxboro service is suspended and doesn't even go via Franklin? (sorry if this is a very obvious question and I'm just not seeing it)

Reply:
Yes, there's no weekday service on the Franklin Line in the Fall schedule. Compare the [spring PDF](https://cdn.mbta.com/sites/default/files/route_pdfs/2020-summer/2020-06-22-cr-franklin-accessible.pdf) to the current [Fall PDF](https://cdn.mbta.com/sites/default/files/route_pdfs/2020-fall/2020-11-02-cr-franklin-accessible-v2.pdf). You can see that Foxboro has dropped out of the list of stops to the left.

Reply:
Got it, thanks for clarifying!